src/main/java/org/example/NumberUtils.java

```diff
@@ -0,0 +1,28 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NumberUtils {
+    public static double getSum(List<Number> numbers) {
+        double sum = 0;
+        for (Number num : numbers) {
+            sum += num.doubleValue();
+        }
+        return sum;
+    }
+
+    public static List<Number> multiplyOddNumber(List<Number> numbers) {
+        List<Number> result = new ArrayList<>();
+        for (Number num : numbers) {
+            int intVal = num.intValue();
+            if (intVal % 2 != 0) {
+                result.add(intVal * 2);
+            } else {
+                result.add(intVal);
+
+            }
+        }
+        return result;
+    }
```

Comment:
Краще використати `long` замість `int` :) В такому випадку Ваш метод зможе корректно опрацювати масив з інтів та лонгів. Також в завданні було вказано що парні числа повинні ігноруватися і не додаватися до нового списку (1, 2, 3 -> 2, 6). У Вашому випадку Ви множите непарні на 2 а парні числа знову додаєте до списку що не є правильним.
```suggestion
    public static List<Number> multiplyOddNumber(List<Number> numbers) {
        List<Number> result = new ArrayList<>();
        for (Number num : numbers) {
            long value = num.longValue();
            if (value % 2 != 0) {
                result.add(value * 2);
            }
        }
        return result;
    }
```